node/config.go

```diff
@@ -78,7 +78,8 @@ type Config struct {
 	RaftLogDir string
 
 	// Configuration of peer-to-peer networking.
-	P2P p2p.Config
+	P2P  p2p.Config
+	QP2P *p2p.Config `toml:",omitempty"`
```

Comment:
preserving indentation of geth

```suggestion
	P2P p2p.Config
	
	// Quorum
	QP2P *p2p.Config `toml:",omitempty"`
```